In [2]:
import numpy as np
import scipy.stats as stats

# Data
durabilities = np.array([1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29])



# a. Build 99% Confidence Interval Using Sample Standard Deviation

In [3]:
# Step 1: Calculate the Sample Mean and Sample Standard Deviation
sample_mean = np.mean(durabilities)
sample_std_dev = np.std(durabilities, ddof=1)
n = len(durabilities)
print(f"Sample Mean: {sample_mean}")
print(f"Sample Standard Deviation: {sample_std_dev}")
print(f"Sample Size: {n}")


Sample Mean: 1.2386666666666666
Sample Standard Deviation: 0.19316412956959936
Sample Size: 15


In [4]:
# Step 2: Use the t-Distribution (since sample std dev is used)
df = n - 1
print(f"df: {df}")
t_critical = stats.t.ppf(0.995, df)  # 99% confidence level, two-tailed test
print(f"t-critical value: {t_critical}")

df: 14
t-critical value: 2.976842734370834


In [5]:
# Step 3: Calculate the Margin of Error (E) for t-distribution
margin_of_error_t = t_critical * (sample_std_dev / np.sqrt(n))
print(f"Margin of Error (E): {margin_of_error_t}")

Margin of Error (E): 0.14846932822817596


In [6]:
# Step 4: Construct the Confidence Interval for t-distribution
ci_lower_t = sample_mean - margin_of_error_t
ci_upper_t = sample_mean + margin_of_error_t
print(f"99% Confidence Interval: ({ci_lower_t}, {ci_upper_t})")

99% Confidence Interval: (1.0901973384384906, 1.3871359948948425)


# b. Build 99% Confidence Interval Using Known Population Standard Deviation

In [7]:
# Step 1: Calculate the Margin of Error (E) for z-distribution (using known population std dev)
population_std_dev = 0.2
z_critical = stats.norm.ppf(0.995)  # 99% confidence level, two-tailed test
print(f"z-critical value: {z_critical}")


z-critical value: 2.5758293035489004


In [8]:
# Step 2: Calculate the Margin of Error (E) for z-distribution
margin_of_error_z = z_critical * (population_std_dev / np.sqrt(n))
print(f"Margin of Error (E): {margin_of_error_z}")



Margin of Error (E): 0.13301525327090588


In [9]:
# Step 3: Construct the Confidence Interval for z-distribution
ci_lower_z = sample_mean - margin_of_error_z
ci_upper_z = sample_mean + margin_of_error_z
print(f"99% Confidence Interval: ({ci_lower_z}, {ci_upper_z})")


99% Confidence Interval: (1.1056514133957607, 1.3716819199375725)



## Analytical Report Conclusion
Based on the given durability data of the samples, the following statistical inferences were made:
Sample Statistics:
Sample Mean (𝑥̄): ≈ 1.25
Sample Standard Deviation (s): ≈ 0.19
Sample Size (n): 15

99% Confidence Interval using t-Distribution (population standard deviation unknown):
The t-critical value (df = 14) is approximately 2.977.
The resulting margin of error is approximately 0.14, giving a 99% confidence interval of:
(1.11, 1.39)

99% Confidence Interval using z-Distribution (population standard deviation = 0.2 known):
The z-critical value is 2.576.
The resulting margin of error is approximately 0.13, giving a 99% confidence interval of:
(1.12, 1.38)

Final Conclusion:
 The material's true mean durability falls between 1.11 and 1.39 (t-interval) or 1.12 and 1.38 (z-interval) at a 99% confidence level.  Both methods yield nearly identical results, indicating that the sample provides a reliable estimate of the population mean.  The consistency between the t- and z-based intervals suggests that the sample size is sufficient and the variation in durability is stable.

